ballerina-shell/modules/shell-cli/src/test/java/io/ballerina/shell/cli/test/validator/ValidatorTest.java

```diff
@@ -34,7 +34,6 @@ public void testValidator() {
         InputValidator inputValidator = new InputValidator();
 
         // Variable declarations
-        Assert.assertTrue(inputValidator.isComplete("int i = 12;"));
```

Comment:
Any specific reason to remove this test?

Reply:
Added missing test case in 1064ce4f587e71a510eba19b47c9dd37ea3da89a